src/main/thrift/parquet.thrift

```diff
@@ -237,6 +237,37 @@ struct SizeStatistics {
    3: optional list<i64> definition_level_histogram;
 }
 
+/**
+ * Bounding box of geometries in the representation of min/max value pair of
+ * coordinates from each axis.
+ */
+struct BoundingBox {
+  /** Min X value when edges = PLANAR, westmost value if edges = SPHERICAL */
```

Comment:
nit: Instead of when/if can we stick to one? Either when/when or if/if would be best.